Fits/Interceptors/[ADV-5] Raptor.md

```diff
@@ -0,0 +1,38 @@
+# Raptor
+
+High DPS Interceptor
+
+
+## Description
+
+A very fast Interceptor focued on burning pings and repositioning, with DPS to target/catch/kill enemy EWAR frigates and long range disruption.
+
+## Fit
+
+```
+[Raptor, [ADV-5] Raptor]
+
+Damage Control II
+Vigor Compact Micro Auxiliary Power Core
```

Comment:
2x Overdrive for all combat interceptors I'd say, we might need to make some sacrifices on PG

Reply:
I just couldn't really figure out their niche without the DPS capability, because if they're just fast then realistically they're just worse versions of the other 4 interceptors since they don't have the increased point/scram range. And they need a bit of tank to actually apply damage to the things they catch. Will recheck them all and see if I can get more speed on them though.